### DISCUSSION TITLE
How to pass additional info in VectorStoreInfo to a LLMChain to have a conversation with a Prompt template & Memory

### DISCUSSION THREAD

**Discussion by astuteprogrammer** [langflow-ai/langflow#379] [Q&A]
For example if I had to provide additional info in a text file through VectoreStoreInfo to the existing sample of "Getting started - Basic Chatbot", how can i achieve it? I see LangChain forums discuss about RetrievalQAWithSourcesChain. I don't see that in LangFlow. Is this a current gap or is there a way to achieve this? I would much appreciate your help. Cheers! 

https://python.langchain.com/en/latest/modules/chains/index_examples/vector_db_qa_with_sources.html

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.
